index.md

```diff
@@ -29,7 +29,7 @@ This code is not exhaustive or complete. It serves to distill our common underst
 
 ### Diversity Statement
 
-We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is equal. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
+We encourage everyone to participate and are committed to building a community for all. Although we may not be able to satisfy everyone, we all agree that everyone is worthy of respect. Whenever a participant has made a mistake, we expect them to take responsibility for it. If someone has been harmed or offended, it is our responsibility to listen carefully and respectfully, and do our best to right the wrong.
```

Comment:
@batjko, The whole point about diversity is to include people who are not equal.

There are many possible sentences that include the word 'equal' I would have no problem with ('equality before the law', 'equal rights', 'equally respected', etc.), but a bare 'everyone is equal' perpetuates the myth that projects are meritocracies, and that all participants (or potential ones) have had the same opportunities, face similar hurdles, etc.